sceptre/scipool/README.md

```diff
@@ -31,15 +31,44 @@ Requirements:
 * Run `pre-commit install` to install the git hook.
 
 ## Testing
+
+### Pre-Commit
 As a pre-deployment step we syntatically validate our sceptre and
 cloudformation yaml files with [pre-commit](https://pre-commit.com).
 
 Please install pre-commit, once installed the file validations will
-automatically run on every commit.  Alternatively you can manually
+automatically run on every commit. Alternatively you can manually
 execute the validations by running `pre-commit run --all-files`.
 Please install pre-commit, once installed the file validations will
 automatically run on every commit.
 
+### Functional Testing
+The process to test the functionality of an AMI and it's integration with our
+Service Catalog products is to first create a test AMI, upload a modified
+product template to S3, and create a new Service Catalog product in the
+scipool dev account to verify manually from https://sc-dev.sageit.org/
+
+The deploy pipelines for both our packer repos and our service catalog library
+will create artifacts for branches that begin with `test/` in the sandbox
+account, allowing anyone with write access to the packer repos to create test
+AMIs, and anyone with write access to service-catalog-library to upload test
+templates for service catalog to S3.
+
+1. Commit changes to the packer repo to update or modify the AMI on a branch
+   that starts with `test/`, and push directly to the origin repo.
```

Comment:
"to the origin repo'" -> "to the origin branch".

Can you please link the words "the packer repo" to the packer repo' URL?

Reply:
The branch name will start with `test/` such as `test/foo`, and that branch is pushed directly to the origin, e.g. `Sage-Bionetworks-IT/packer-ami-template`. I've reworded this to 'upstream repo' with an explicit example.